test/test_ops.py

```diff
@@ -2039,6 +2067,18 @@ def test_interpolate_bilinear(self):
       lambda x: torch.nn.functional.interpolate((10*x).relu().type(torch.uint8), size=out_sz, mode="bilinear"),
       lambda x: Tensor.interpolate((10*x).relu().cast('uint8'), size=out_sz, mode="linear"), forward_only=True)
 
+  def test_interpolate_nearest(self):
```

Comment:
Does the bilinear test above work? It's still skipped

Reply:
it doesn't work yet. 
I'll try to fix this too.

Reply:
it seems that there are uint8 specific interpolation kernels for pytorch.
I think I've pinned down the difference that's causing numerical issues to this
https://github.com/pytorch/pytorch/blob/2bd02e0c82c6d76157244655256062bade5070ae/aten/src/ATen/native/cpu/UpSampleKernel.cpp#L943
I don't 100% get what's going on, so I'm gonna stare at it for a bit longer until I figure it out

also apparently there is also some lerp numerical instability as well.
https://github.com/pytorch/pytorch/pull/18871
Though I copied their impl and it didn't fix the bilinear test.